internal/bitwarden/bw/models.go

```diff
@@ -90,21 +91,25 @@ type SecureNote struct {
 }
 
 type Object struct {
-	CollectionIds  []string   `json:"collectionIds,omitempty"`
-	ID             string     `json:"id,omitempty"`
-	ExternalID     string     `json:"externalId,omitempty"`
-	FolderID       string     `json:"folderId,omitempty"`
-	Login          Login      `json:"login,omitempty"`
-	Name           string     `json:"name,omitempty"`
-	Notes          string     `json:"notes,omitempty"`
-	Object         ObjectType `json:"object,omitempty"`
-	OrganizationID string     `json:"organizationId,omitempty"`
-	SecureNote     SecureNote `json:"secureNote,omitempty"`
-	Type           ItemType   `json:"type,omitempty"`
-	Fields         []Field    `json:"fields,omitempty"`
-	Reprompt       int        `json:"reprompt,omitempty"`
-	Favorite       bool       `json:"favorite,omitempty"`
-	RevisionDate   *time.Time `json:"revisionDate,omitempty"`
+	CollectionIds  []string     `json:"collectionIds,omitempty"`
+	ID             string       `json:"id,omitempty"`
+	ExternalID     string       `json:"externalId,omitempty"`
+	FolderID       string       `json:"folderId,omitempty"`
+	Login          Login        `json:"login,omitempty"`
+	Name           string       `json:"name,omitempty"`
+	Notes          string       `json:"notes,omitempty"`
+	Object         ObjectType   `json:"object,omitempty"`
+	OrganizationID string       `json:"organizationId,omitempty"`
+	SecureNote     SecureNote   `json:"secureNote,omitempty"`
+	Type           ItemType     `json:"type,omitempty"`
+	Fields         []Field      `json:"fields,omitempty"`
+	Reprompt       int          `json:"reprompt,omitempty"`
+	Favorite       bool         `json:"favorite,omitempty"`
+	RevisionDate   *time.Time   `json:"revisionDate,omitempty"`
+	File           string       `json:"file,omitempty"`
```

Comment:
As explained in my feedback, I don't think we should put that here. It's a bit weird to have attachment attributes in the structure that holds Attachments.